testgrid/specs/full.yaml

```diff
@@ -41,14 +41,14 @@
     minio:
       version: latest
   airgap: true
-- name: k8s121x_docker19034
+- name: k8s121x_docker201017
```

Comment:
Modified full.yaml to use 20.10.17 everywhere instead of 19.03.4. If things do not work in staging, we need to revert.

Reply:
Leaving the pinned docker versions as is